draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1195,29 +1202,29 @@ uCDN-A                         dCDN-B                         dCDN-C
    Description: The time at which the trigger resource was received by the dCDN. 
    The time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
+   Value: Time, as defined in Section 4.3.4 of <xref target="RFC8006"/>.
    <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Mandatory: dCDN MUST specify in trigger status representations. It is ignored when included in trigger representations sent by uCDN.
    </t>
 
    <t hangText="Name: mtime">
    <vspace blankLines="1"/>
    Description: The time at which the trigger resource was last modified. 
    The time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
+   Value: Time, as defined in Section 4.3.4 of <xref target="RFC8006"/>.
    <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Mandatory: dCDN MUST specify in trigger status representations. It is ignored when included in trigger representations sent by uCDN.
    </t>
 
    <t hangText="Name: etime">
    <vspace blankLines="1"/>
    Description: The estimate of the time at which dCDN expects to complete the trigger processing. 
    Time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
+   Value: Time, as defined in Section 4.3.4 of <xref target="RFC8006"/>.
    <vspace blankLines="1"/>
-   Mandatory: No.
+   Mandatory: dCDN MUST specify in trigger status representations. It is ignored when included in trigger representations sent by uCDN.
```

Comment:
This one was technically not mandatory, so should we change the language from MUST to SHOULD or MAY?  Or do you think we should make it mandatory?